exs/mundo_1/ruby/001.rb

```diff
@@ -0,0 +1,7 @@
+=begin
+Desafio 001
+Problema: Crie um programa que escreva "Olá mundo" na tela.
+Resolução do problema:
+=end
```

Comment:
Mantenha o espaçamento no enunciado, seguindo o padrão dos demais

```suggestion
=begin
Desafio 001

Problema: Crie um programa que escreva "Olá mundo" na tela.

Resolução do problema:
=end
```